### ISSUE TITLE
Vanilla maps don't work

### ISSUE THREAD

**Issue by MrDuq** [zDevelopers/ImageOnMap#99]
Treasure maps, Ocean explorer, woodland explorer and vanilla maps give ImageOnMap maps:

![example](https://user-images.githubusercontent.com/67129259/85084455-d5eb3000-b1a2-11ea-85d5-210ab26e7ff8.gif)

**Comment by Vlammar** [comment]
Hello, is it possible to get a copy of your minecraft server ? I tried to replicate what you did on your gif and it work perfectly fine . You are running the 4.0 version ? Are you on paper or spigot ?

**Comment by MrDuq** [comment]
Hey, so I was able to fix the bug by deleting some large and unused maps in the maps folder, I think we just maxed out the allowed number of maps but I’m not sure. Also we’re on paper.

**Comment by AmauryCarrade** [comment]
The max number of maps is over two **billion**, so I doubt it… Well, I guess we'll see if this is reported again?

**Comment by MrDuq** [comment]
Wow ok, we only had about 3000 I have a backup of when the bug was still happening, (there's some stupid maps lol) 
 [https://drive.google.com/file/d/1K6gzYL5cpvQJ9xIsFW070DHy8zeOxxXl/view?usp=sharing]

Edit: sorry it took so long by the way, my friend has a server pc that he runs it off so I had to wait for him to send the file.

**Comment by Vlammar** [comment]
Any update ?

**Comment by MrDuq** [comment]
https://drive.google.com/file/d/1K6gzYL5cpvQJ9xIsFW070DHy8zeOxxXl/view?usp=sharing I forgot to post the link I guess

**Comment by Vlammar** [comment]
I need your autorization to access your drive

**Comment by MrDuq** [comment]
Oh, sorry, should be accessible now.

**Comment by Vlammar** [comment]
Did you removed map.dat files ? I've only got 7 map.dat files so when bukkit give a mapid it give an already used Id

**Comment by MrDuq** [comment]
No, that’s probably what it was. I just deleted some unused maps and that fixed it.

**Comment by Vlammar** [comment]
> 
> 
> No, that’s probably what it was. I just deleted some unused maps and that fixed it.

What annoys me is bukkit should give you unused map id and for a strange reason you only got like 7 maps that's not possible because you got like more than 1000 images in the image folder. How did you delete those maps ?

**Comment by MrDuq** [comment]
I don’t fully understand the question but I had the server owner give me access via team-viewer, then I went to Plugins > ImageOnMap > images and deleted a bunch of them by just selecting them and pressing del key.

**Comment by KingsRequin** [comment]
hello, I have exactly the same problem with vanilla maps..
I have to open a new issue?

**Comment by Vlammar** [comment]
Those bugs comes from a corruption of the playermapstore i'm working on a fix but i'm not sure what caused this corruption

**Comment by KingsRequin** [comment]
okay, thank you and good luck!
